lib/hanami/cli.rb

```diff
@@ -88,16 +88,15 @@ def console
     desc 'new PROJECT_NAME', 'Generate a new hanami project'
     long_desc <<-EOS
 `hanami new` creates a new hanami project.
-You can specify various options such as the database to be used as well as the path and architecture.
+You can specify various options, like which database type, testing framework or templating engine the project will use.
 
 $ > hanami new fancy_app --application_name=admin
 
-$ > hanami new fancy_app --arch=app
+$ > hanami new fancy_app --database=postgresql
 
 $ > hanami new fancy_app --hanami-head=true
     EOS
     method_option :database, aliases: ['-d', '--db'], desc: "Application database (#{Hanami::Generators::DatabaseConfig::SUPPORTED_ENGINES.keys.join('/')})", default: Hanami::Generators::DatabaseConfig::DEFAULT_ENGINE
-    method_option :architecture, aliases: ['-a', '--arch'], desc: 'Project architecture (container/app)', default: Hanami::Commands::New::Abstract::DEFAULT_ARCHITECTURE
     method_option :application_name, desc: 'Application name, only for container', default: Hanami::Commands::New::Container::DEFAULT_APPLICATION_NAME
```

Comment:
As there are no more non-container arch, it worth to leave documentation of this options simply:

``` ruby
method_option :application_name, desc: 'Application name'. default: ...
```